pkg/azureutils/azure_disk_utils.go

```diff
@@ -84,6 +84,25 @@ const (
 	diskNameGenerateMaxLength = 76
 )
 
+var AttachmentRoles = map[AttachmentRoleMode]string{
+	PrimaryOnly: string(azdiskv1beta2.PrimaryRole),
+	ReplicaOnly: string(azdiskv1beta2.ReplicaRole),
+}
+
+type AttachmentRoleMode int
+
+const (
```

Comment:
Would be good to combine this with the rest of the constants at the beginning of the file.

Reply:
I believe defining these constant after its type definition makes it more readable.